backend/src/game/PokerTable.test.ts

```diff
@@ -92,4 +92,44 @@ describe('PokerTable', () => {
       expect(pokerTable?.getName()).toEqual('table_1');
     });
   });
+  describe('leaveTable', () => {
+    it('should confirm a player has left the table', () => {
+      const tableName = 'table_1';
+      GameLobbyService.createPokerTable(tableName, 2);
+      jest.spyOn(Rooms, 'sendEventToRoom').mockImplementation(() => Result.success());
+      const pokerTable = GameLobbyService.getTable(tableName);
+  
+      if (!pokerTable) {
+        // TODO: log error
+        return;
+      }
+      pokerTable.sitAtTable(tableName, 'seat-1', 'a1')
+      const res = pokerTable.leaveTable('seat-1', 'a1')
+  
+      expect(res.ok).toEqual(true);
```

Comment:
We actually want to also check that the pokerTable has that seat free too. So we should really have another function on PokerTable: `getAvailableSeats` or something which would probably come in use regardless but would be useful in this test to check that the pokerTable's seat was free'd up.